Sources/PiholeNetworking/PHProvider+Experimental.swift

```diff
@@ -19,7 +19,7 @@ extension PHProvider {
 	/// - Parameter from: Start date. This will be rounded down to the nearest multiple of the interval
 	/// - Parameter until: End date. The last interval returned will always be less than this date rounded down to the nearest second
 	/// - Parameter interval: The size of each reported interval, which will be rounded down to the nearest second
-	/// - Returns: Ad and domain query counts in the interval [start date, end date)
```

Comment:
This was supposed to be a math syntax interval but I guess I'm too much of a nerd
Math notation for a half open interval

Reply:
It was supposed to be clear that you can request a->b, and then b->c without having to worry about getting whatever is at b twice. No doing weird things like a->b, b+0.01->c